rpc/documentation/public_endpoints/getnetworkgraph.md

```diff
@@ -13,6 +13,8 @@ None
 | `density`                            | f64        | The density of the known network                                                          |
 | `algebraic_connectivity`             | f64        | The algebraic connectivity, aka the fiedler eigenvalue of the known network               |
 | `degree_centrality_delta`            | f64        | The difference between the largest and the smallest peer count in the network             |
+| `potential_tip`                      | Option     | The crawler's best guess at the canon tip                                                 |
```

Comment:
```suggestion
| `potential_tip`                      | Option     | The crawler's best guess at the canon block height                                        |
```

Reply:
This doesn't only contain the block height but also the member nodes, I'll make this explicit though 👍